setup.py

```diff
@@ -96,7 +96,7 @@
     # For Python 2.6 and 2.7, any version *except* 2.0.0 will work
     install_requires = ['pyparsing!=2.0.0']
 
-install_requires += ['astropy', 'numpy', 'Cython']
+install_requires += ['astropy>=1.0.0', 'numpy', 'Cython']
```

Comment:
Maybe just put `astropy>=1.0` here instead of `1.0.0`?

At  https://travis-ci.org/astropy/pyregion/jobs/151599925#L639 I see that we're testing against Astropy 1.0.10, so we don't really know if any bugs that were fixed since Astropy 1.0.0 affect pyregion.